presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -105,6 +105,42 @@ General Aggregate Functions
 
     Returns ``n`` smallest values of all input values of ``x``.
 
+.. function:: reduce_agg(inputValue T, initialState S, inputFunction(S, T, S), combineFunction(S, S, S)) -> S
+
+    Returns a single value reduced from input values. ``inputFunction`` and ``combineFunction`` will be invoked.
```

Comment:
This is a bit wide. We should wrap at 80-100 characters.

It seems strange to say "will be invoked" as a separate sentence. The other docs say that as part of the description. How about

```
Reduces all input values into a single value. ```inputFunction`` will be invoked
for each input value. In addition to taking the input value, ``inputFunction``
takes the current state, initially ``initialState``, and returns the new state.
``combineFunction`` will be invoked to combine two states into a new state.
The final state is returned::

Reply:
I changed the description to be very similar to that of `reduce`, since we spent a while coming up with a good description for that.